### PR TITLE
temporarily disable weather, fix station id issue

### PR DESCRIPTION
**Pull request by zhawan**
temporarily disable weather, fix station id issue

### CODE REVIEW

File: maro/cli/data_pipeline/citi_bike.py [hunk 1]
```diff
@@ -364,7 +364,8 @@ class CitiBikeTopology(DataTopology):
     def __init__(self, topology: str, trip_source: str, station_info: str, weather_source: str, is_temp: bool = False):
         super().__init__()
         self._data_pipeline["trip"] = CitiBikePipeline(topology, trip_source, station_info, is_temp)
-        self._data_pipeline["weather"] = WeatherPipeline(topology, weather_source, is_temp)
+        # Weather data source changed, temporarily disable, will enable it later when new data source is available.
```
Comment (ArthurJiang) [inline maro/cli/data_pipeline/citi_bike.py:367]: # TODO: ...

File: maro/simulator/scenarios/citi_bike/station.py [hunk 2]
```diff
@@ -48,6 +51,7 @@ def reset(self):
         self.capacity = self._init_capacity
         self.bikes = self._init_bikes
         self.min_bikes = self._init_bikes
+        self.id = self._id
```
Comment (ArthurJiang) [inline maro/simulator/scenarios/citi_bike/station.py:55]: Do we need rm weather in the station?

Comment (ArthurJiang) [reply maro/simulator/scenarios/citi_bike/station.py:55]: @chaosddp

Comment (chaosddp) [reply maro/simulator/scenarios/citi_bike/station.py:55]: at least remove weather reader from be,  or not exist file path may cause error

**Pull request merged**